test/Numerics/ODESolvers/ode_tests_convergence.jl

```diff
@@ -669,7 +707,7 @@ const ArrayType = ClimateMachine.array_type()
                 ArrayType([sqrt(3 + cos(ω * t)); sqrt(2 + cos(t))])
 
             finaltime = 1
-            dts = [2.0^(-k) for k in 6:7]
+            dts = [2.0^(-k) for k in 6:8]
```

Comment:
Why the increase?

Reply:
`Trap2` needed an additional nudge to reach the `atol` criteria in its convergence check